mica-webapp/src/main/resources/_templates/libs/study.ftl

```diff
@@ -11,7 +11,7 @@
             <@studyDesign study=study/>
           </#if>
 
-          <#if study.model.markerPaper??>
+          <#if study.model.markerPaper?has_content>
```

Comment:
If it is null,it will fail.

Reply:
I tried with NULL and removed the field from the model and never got an error. I think if the field is null or does not exists _has_content_ returns false: [has_content](https://freemarker.apache.org/docs/ref_builtins_expert.html#ref_builtin_has_content)